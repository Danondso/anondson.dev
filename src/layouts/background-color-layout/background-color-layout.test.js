import React from "react"
import renderer from "react-test-renderer"
import BackgroundColorLayout from "./background-color-layout"

describe("BackgroundColorLayout Snapshot test", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <BackgroundColorLayout
          colorName="#000000"
          altName="Test Background Section"
        >
          {" "}
          <h1>Hello</h1>{" "}
        </BackgroundColorLayout>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
