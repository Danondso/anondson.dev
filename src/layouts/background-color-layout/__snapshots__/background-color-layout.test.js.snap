// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackgroundColorLayout Snapshot test renders correctly 1`] = `
<section
  alt="Test Background Section"
  style={
    Object {
      "backgroundColor": "#000000",
      "boxShadow": "0 3px 2px rgba(0, 0, 0, 0.15)",
      "padding": "4rem",
    }
  }
>
   
  <h1>
    Hello
  </h1>
   
</section>
`;
